clab/config/functions.go

```diff
@@ -0,0 +1,230 @@
+package config
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"inet.af/netaddr"
+)
+
+func typeof(val interface{}) string {
+	switch val.(type) {
+	case string:
+		return "string"
+	case int, int16, int32:
+		return "int"
+	}
+	return ""
+}
+
+func hasInt(val interface{}) (int, bool) {
```

Comment:
I wonder why the func is named `hasInt` and not `isInt`?

Reply:
It can even be `parseInt` ;-)

Reply:
how do you use this func in the templates? What inputs you feed to it?
 Just want to understand it better

Reply:
it receives any unknown type (`interface{}`) where we expect an `int`. So this could be a real `int` (mostly unlikely since our "labels" are strings) or it could be a `string` containing a number.

The function then returns this int and a bool to indicate if it was really an int

Reply:
got it. then to me `toInt` makes a nice name for that func